package.json

```diff
@@ -12,15 +16,21 @@
   "dependencies": {
     "@rails/webpacker": "3.5",
     "babel-preset-react": "^6.24.1",
+    "jquery": "3.3.1",
+    "lodash": "4.17.11",
     "prop-types": "^15.6.1",
     "react": "^16.4.1",
+    "react-bootstrap": "0.32.4",
     "react-dom": "^16.4.1",
+    "react-select2-wrapper": "cooperka/react-select2-wrapper#2019-03-08-select2Full",
```

Comment:
I forked it to allow using `select2.full.js` which is needed for using config such as `dropdownCssClass`; if you want the fork to be under TCC let me know.

So far no reply ([PR here](https://github.com/rkit/react-select2-wrapper/pull/89)).

Reply:
The other forks are under sassafras so let's put it there.

I forked to https://github.com/sassafrastech/react-select2-wrapper and gave you access.